components/ui/colors/src/main/java/mozilla/components/ui/colors/PhotonColors.kt

```diff
@@ -33,6 +33,7 @@ object PhotonColors {
     val Green80 = Color(0xff006504)
     val Green90 = Color(0xff003706)
 
+    val Red30 = Color(0xffff848b)
```

Comment:
Let's also add this to our existing colors.xml in case or maybe at least until we can deprecate the xml?

Reply:
This is done in the first PR.